config/routes.rb

```diff
@@ -443,6 +443,7 @@
 
           resource :mute, only: :create
           post :unmute, to: 'mutes#destroy'
+          post :clear_notifications, to: 'mutes#clear_notifications'
```

Comment:
Putting that in the `MutesController` sound a bit weird tbh, but I'm not sure where it would fit best.

Reply:
yeah same, felt it was sort of related to muting and didn't think it was enough for its own controller